### ISSUE TITLE
[1.16.1] Reproducible crash when trashing an item

### ISSUE THREAD

**Issue by Protected** [blay09/TrashSlot#81]
TrashSlot: 12.1.0
Forge: 32.0.108

When I attempt to drag an item or block (you can test with a block of dirt) into the slot, the screen will freeze and minecraft will crash immediately. Test performed using a freshly made single player world. Other relevant mods are listed in the exception, but there are no other known crashes in this setup. I'll add more details as I figure them out.

https://gist.github.com/Protected/744697457aa9d285a53f548ee8128968

**Comment by Protected** [comment]
I have disabled every mod listed in the stack trace as having registered a listener here, and the crash still takes place.

**Comment by Protected** [comment]
I noticed I was incorrectly using the version for 1.16.2, so I made sure to test with 12.0.0 and 12.0.1 as well. 12.0.0 has a similar crash:

java.lang.NoSuchMethodError: net.minecraft.client.gui.screen.Screen.func_238654_b_(Lcom/mojang/blaze3d/matrix/MatrixStack;Ljava/util/List;IILnet/minecraft/client/gui/FontRenderer;)V
	at net.blay09.mods.trashslot.client.gui.GuiHelper.renderTooltip(GuiHelper.java:24)
...etc

but 12.0.1 is working. Sorry about that!
